habitat_baselines/rl/models/rnn_state_encoder.py

```diff
@@ -376,7 +376,7 @@ def unpack_hidden(
         self, hidden_states
     ) -> Tuple[torch.Tensor, torch.Tensor]:
         lstm_states = torch.chunk(hidden_states, 2, 0)
-        return (lstm_states[0], lstm_states[1])
+        return (lstm_states[0].contiguous(), lstm_states[1].contiguous())
```

Comment:
Actually, shouldn't the chunks still be contiguou assuming the underlying hidden_state is? We may be able to remove this call if we figure out where the non-contiguous hidden state is stored.